### PR TITLE
:sparkles: add support for workflows

### PR DESCRIPTION
**Pull request by sebastianMindee**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires a change to the official Guide documentation.

### CODE REVIEW

File: docs/extras/code_samples/workflow_execution.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Load a file from disk
+input_doc = mindee_client.source_from_path("/path/to/the/file.ext")
+
+# Send the file to the workflow.
+result: WorkflowResponse = mindee_client.execute_workflow(
+    input_doc,
+    workflow_id,
+    # Optionally, add an alias and a priority to the workflow.
+    # alias="my-alias",
+    # priority=ExecutionPriority.LOW
```
Comment (ianardee) [inline docs/extras/code_samples/workflow_execution.txt:17]: import the `ExecutionPriority` class?

**Review by ianardee — CHANGES_REQUESTED** [review]

File: mindee/client.py [hunk 2]
```diff
@@ -230,6 +235,42 @@ def parse_queued(
...
+        self,
+        input_source: Union[LocalInputSource, UrlInputSource],
+        workflow_id: str,
+        page_options: Optional[PageOptions] = None,
+        alias: Optional[str] = None,
+        priority: Optional[ExecutionPriority] = None,
+        full_text: bool = False,
+    ) -> WorkflowResponse:
+        """
+        Send the document to an asynchronous endpoint and return its ID in the queue.
```
[the remark this answers is not in the document]
Comment (sebastianMindee) [reply mindee/client.py:246]: 🤭

File: docs/extras/code_samples/workflow_execution.txt (hunk 1 above)
Comment (sebastianMindee) [reply docs/extras/code_samples/workflow_execution.txt:17]: Well, it's commented out so I reasoned it wouldn't be necessary, but sure :+1:

Comment (ianardee) [reply docs/extras/code_samples/workflow_execution.txt:18]: was more of a "do you think we should?", but yes, better I think ;-)

**Review by ianardee — APPROVED** [review]

**Pull request merged**